rotkehlchen/tests/unit/decoders/test_makerdao.py

```diff
@@ -52,12 +49,128 @@ def test_makerdao_simple_transaction(
+            location_label=user_address,
+            notes='Burned 0.001301015216220134 ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_SDAI,
+            balance=Balance(amount=FVal('16.020774067834506624')),
+            location_label=user_address,
+            notes='Return 16.020774067834506624 sDAI to sDAI contract',
+            tx_hash=tx_hash,
+            counterparty=CPT_SDAI,
+            address=sdai.evm_address,
+        ), EvmEvent(
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=A_DAI,
+            balance=Balance(amount=FVal('16.601085935411927527')),
+            location_label=user_address,
+            notes='Withdraw 16.601085935411927527 DAI from sDAI contract',
+            tx_hash=tx_hash,
+            counterparty=CPT_SDAI,
+            address=sdai.evm_address,
+        ),
+    ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xa217BDa86b0EDb86eE7d4D6e34F493eDF1ea4F29']])  # noqa: E501
+def test_deposit_dai_to_sdai(
+        database,
+        ethereum_inquirer,
+        ethereum_accounts,
+):
+    user_address = ethereum_accounts[0]
+    tx_hash = deserialize_evm_tx_hash('0x27bd72a2ccd999a44c2a7aaed9090572f34045d62e153362a34715a70ca7a6a7')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1695089927000)
+    assert events == [
+        EvmEvent(
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(0.00152049387145495)),
+            location_label=user_address,
+            notes='Burned 0.00152049387145495 ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_DAI,
+            balance=Balance(amount=FVal(16.58145794)),
+            location_label=user_address,
+            notes='Deposit 16.58145794 DAI to sDAI contract',
+            tx_hash=tx_hash,
+            counterparty=CPT_SDAI,
+            address=A_SDAI.resolve_to_evm_token().evm_address,
```

Comment:
just resolve its address or the token itself in the top of the test and re-use it here. Don't resolve multiple times in the test